docs/SETTINGS.md

```diff
@@ -4,6 +4,18 @@
 
 The language server supports the following configuration options:
 
+## `terraformLogFilePath` (`string`)
+
+Path to a file for Terraform executions to be logged into (TF_LOG_PATH)
```

Comment:
@radeksimko I've used the same desc from the client side changes, If you follow a particular approach for user facing strings, kindly suggest the same. Thanks.